### PR TITLE
New package: variety-0.6.0

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: srcpkgs/variety/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'variety'
+pkgname=variety
+version=0.6.0
+revision=1
+wrksrc="variety"
+hostmakedepends="python-distutils-extra intltool"
+depends="desktop-file-utils gtk+3 ImageMagick libnotify python-BeautifulSoup4
+	python-cairo python-configobj python-dbus exiv2 python-gobject python-httplib2
```
Comment (Duncaen) [inline srcpkgs/variety/template]: Use one space for indention inside variables.

File: srcpkgs/variety/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	python-Pillow python-lxml python-curl webkitgtk"
+short_desc="Changes the wallpaper on a regular interval"
+maintainer="Diogo Leal <diogo@diogoleal.com>"
+license="GPL-3"
+homepage="https://launchpad.net/variety/"
+distfiles="https://launchpad.net/variety/trunk/${version}/+download/variety_${version}.tar.gz"
+checksum=f3760ac546073bb68f840f5eafffaf6a265e5be39aaa36462ff74547fe0594d0
+
+do_install() {
+	python setup.py install --root="${DESTDIR}"
```
Comment (Duncaen) [inline srcpkgs/variety/template]: Why dont you use the `python-module` build_style?

Comment (diogoleal) [inline srcpkgs/variety/template]: When I use python-module, this error occurs:
```
=> variety-0.6.0_1: running do_install ...
ERROR: Python module varietyslideshow not found
ERROR: Python module configobj not found
ERROR: Python module configobj not found
ERROR: Python module helpers not found
ERROR: Python module varietyconfig not found
ERROR: Python module bs4 not found
ERROR: Python module helpers not found
ERROR: Python module IVarietyPlugin not found
ERROR: Python module PIL not found
ERROR: Python module helpers not found
ERROR: Python module varietyconfig not found
ERROR: Python module varietyslideshow not found
ERROR: Python module bs4 not found
ERROR: Python module pyexiv2 not found
ERROR: Python module DominantColors not found
ERROR: Python module bs4 not found
ERROR: Python module pyexiv2 not found
ERROR: Python module DominantColors not found
ERROR: Python module configobj not found
ERROR: Python module dbus not found
ERROR: Python module dbus.service not found
ERROR: Python module dbus.glib not found
ERROR: Python module IPlugin not found
ERROR: Python module DominantColors not found
ERROR: Python module configobj not found
ERROR: Python module configobj not found
ERROR: Python module helpers not found
ERROR: Python module PIL not found
ERROR: Python module lxml not found
ERROR: Python module helpers not found
ERROR: Python module Util not found
ERROR: Python module cairo not found
ERROR: Python module PIL not found
ERROR: Python module bs4 not found
ERROR: Python module PIL not found
ERROR: Python module pycurl not found
ERROR: Python module Util not found
ERROR: Python module varietyslideshow not found
ERROR: Python module DominantColors not found
ERROR: Python module dbus not found
ERROR: Python module dbus.service not found
ERROR: Python module dbus.glib not found
ERROR: Python module PIL not found
ERROR: Python module varietyconfig not found
ERROR: Python module Builder not found
ERROR: Python module pycurl not found
ERROR: Python module PIL not found
ERROR: Python module varietyconfig not found
ERROR: Python module Builder not found
ERROR: Python module configobj not found
ERROR: Python module IVarietyPlugin not found
ERROR: Python module cairo not found
ERROR: Python module PIL not found
ERROR: Python module helpers not found
ERROR: Python module varietyslideshow not found
ERROR: Python module lxml not found
ERROR: Python module PIL not found
Traceback (most recent call last):
  File "setup.py", line 87, in <module>
    cmdclass={'install': InstallAndUpdateDataDirectory}
  File "/usr/lib/python2.7/site-packages/DistUtilsExtra/auto.py", line 100, in setup
    distutils.core.setup(**attrs)
  File "/usr/lib/python2.7/distutils/core.py", line 111, in setup
    _setup_distribution = dist = klass(attrs)
  File "/usr/lib/python2.7/distutils/dist.py", line 259, in __init__
    getattr(self.metadata, "set_" + key)(val)
  File "/usr/lib/python2.7/distutils/dist.py", line 1230, in set_provides
    distutils.versionpredicate.split_provision(v)
  File "/usr/lib/python2.7/distutils/versionpredicate.py", line 160, in split_provision
    raise ValueError("illegal provides specification: %r" % value)
ValueError: illegal provides specification: 'build-2.7.lib.jumble'
=> ERROR: variety-0.6.0_1: failed to run do_install() at line 49.
```

Comment (Vaelatern) [inline srcpkgs/variety/template]: Perhaps you should hostmakedepends on `python-setuptools` and try again?

**Comment by diogoleal** [comment]
even adding:
```
build_style=python-module
pycompile_module="variety"
hostmakedepends="python-setuptools python-distutils-extra intltool" 
depends="desktop-file-utils gtk+3 ImageMagick libnotify python-BeautifulSoup4
        python-cairo python-configobj python-dbus exiv2 python-gobject python-httplib2
        python-Pillow python-lxml python-curl webkitgtk"
```
Thoses errors occurs:
```
msgfmt po/ru.po -o build/mo/ru/LC_MESSAGES/variety.mo
intltool-merge -d po variety.desktop.in build/share/applications/variety.desktop
Merging translations into build/share/applications/variety.desktop.
running build_icons
running build_help
WARNING: the following files are not recognized by DistUtilsExtra.auto:
  Variety.iml
=> variety-0.6.0_1: running pre-install hook: 00-lib32 ...
=> variety-0.6.0_1: running pre-install hook: 02-script-wrapper ...
=> variety-0.6.0_1: running do_install ...
ERROR: Python module varietyslideshow not found
ERROR: Python module configobj not found
ERROR: Python module configobj not found
ERROR: Python module helpers not found
ERROR: Python module varietyconfig not found
ERROR: Python module bs4 not found
ERROR: Python module helpers not found
ERROR: Python module IVarietyPlugin not found
ERROR: Python module PIL not found
ERROR: Python module helpers not found
ERROR: Python module varietyconfig not found
ERROR: Python module varietyslideshow not found
ERROR: Python module bs4 not found
ERROR: Python module pyexiv2 not found
ERROR: Python module DominantColors not found
ERROR: Python module bs4 not found
ERROR: Python module pyexiv2 not found
ERROR: Python module DominantColors not found
ERROR: Python module configobj not found
ERROR: Python module dbus not found
ERROR: Python module dbus.service not found
ERROR: Python module dbus.glib not found
ERROR: Python module IPlugin not found
ERROR: Python module DominantColors not found
ERROR: Python module configobj not found
ERROR: Python module configobj not found
ERROR: Python module helpers not found
ERROR: Python module PIL not found
ERROR: Python module lxml not found
ERROR: Python module helpers not found
ERROR: Python module Util not found
ERROR: Python module cairo not found
ERROR: Python module PIL not found
ERROR: Python module bs4 not found
ERROR: Python module PIL not found
ERROR: Python module pycurl not found
ERROR: Python module Util not found
ERROR: Python module varietyslideshow not found
ERROR: Python module DominantColors not found
ERROR: Python module dbus not found
ERROR: Python module dbus.service not found
ERROR: Python module dbus.glib not found
ERROR: Python module PIL not found
ERROR: Python module varietyconfig not found
ERROR: Python module Builder not found
ERROR: Python module pycurl not found
ERROR: Python module PIL not found
ERROR: Python module varietyconfig not found
ERROR: Python module Builder not found
ERROR: Python module configobj not found
ERROR: Python module IVarietyPlugin not found
ERROR: Python module cairo not found
ERROR: Python module PIL not found
ERROR: Python module helpers not found
ERROR: Python module varietyslideshow not found
ERROR: Python module lxml not found
ERROR: Python module PIL not found
Traceback (most recent call last):
  File "setup.py", line 87, in <module>
    cmdclass={'install': InstallAndUpdateDataDirectory}
  File "/usr/lib/python2.7/site-packages/DistUtilsExtra/auto.py", line 100, in setup
    distutils.core.setup(**attrs)
  File "/usr/lib/python2.7/distutils/core.py", line 111, in setup
    _setup_distribution = dist = klass(attrs)
  File "/usr/lib/python2.7/distutils/dist.py", line 259, in __init__
    getattr(self.metadata, "set_" + key)(val)
  File "/usr/lib/python2.7/distutils/dist.py", line 1230, in set_provides
    distutils.versionpredicate.split_provision(v)
  File "/usr/lib/python2.7/distutils/versionpredicate.py", line 160, in split_provision
    raise ValueError("illegal provides specification: %r" % value)
ValueError: illegal provides specification: 'build-2.7.lib.jumble'
=> ERROR: variety-0.6.0_1: failed to run do_install() at line 49.
```

File: srcpkgs/variety/template (hunk 2 above)
Comment (Vaelatern) [inline srcpkgs/variety/template]:
```
noarch=yes
build_style=python-module
pycompile_module='variety'
do_build() {
	:
}
```

Comment (Vaelatern) [inline srcpkgs/variety/template]: And delete the `do_install()`

Comment (diogoleal) [inline srcpkgs/variety/template]: done and it's OK, but I don't understand the ":" in function do_build().

Comment (Vaelatern) [inline srcpkgs/variety/template]: It's a `noop`

**Pull request merged**